test/test.code-workspace

```diff
@@ -0,0 +1,10 @@
+{
+    "folders": [
+        {
+            "path": "D:/testOutput"
```

Comment:
This will have to change as well.  It might be harder since it's settings file, but I think you can do dynamic vscode settings.  Here are some references: https://code.visualstudio.com/docs/editor/variables-reference